lib/hushed/blackboard.rb

```diff
@@ -40,5 +42,40 @@ def build_document(type, contents)
       end
       namespace.const_get(type).new(:io => contents) if namespace
     end
+
+    def retrieve_latest(document_name)
+      blackboard = client.from_quiet_bucket
+      documents = inventory_summary_documents(blackboard, document_name)
+      get_latest_summary_report(documents)
+    end
+
+    private
+
+    def inventory_summary_documents(blackboard, doc_name)
+      doc_type = Regexp.new('\A' + doc_name + '(...)').freeze
```

Comment:
> Some people, when confronted with a problem, think "I know, I'll use regular expressions." Now they have two problems.
Regex pretty much always gives me headache.  I feel that start_with? is sufficient and simpler.
```
if document.key.start_with?(doc_type)
    date_string = get_date_string(document.key)
    date = Hushed::DateService.new.build_date(date_string)
    summary_documents[date] = document
end
```